R/convert_stat_chisq.R

```diff
@@ -27,8 +27,9 @@
 #' \cr
 #' For adjusted versions, see Bergsma, 2013.
 #'
-#' @inheritSection effectsize-CIs Confidence Intervals
-#' @inheritSection effectsize-CIs CI Contains Zero
+#' @inheritSection effectsize_CIs Confidence (Compatibility) Intervals (CIs)
+#' @inheritSection effectsize_CIs CIs and Significance Tests
+#' @inheritSection effectsize_CIs One-Sided CIs
```

Comment:
@bwiernik I didn't originally add these as this seemed to much make the individual docs way too long for my liking. We can reference this and the troubleshooting in the same line.

Reply:
As I have placed, say, here:

https://github.com/easystats/effectsize/pull/366/files?file-filters%5B%5D=.Rd&hide-deleted-files=true#diff-e3faf4c1fa910d1c629afa2d7d19afa68c4a54b02237e0f21cd792a15d177c53R89

Reply:
Fine by me.